foundations/introduction/join_the_odin_community.md

```diff
@@ -66,7 +66,7 @@ Your Multiple Lines of Code
 
 You can also use _code highlighting_ to add color to your multi-line code by specifying the language:
 
-\`\`\`**javascript**
+\`\`\`**js** or **javascript**
```

Comment:
I changed it to list 'js' first rather than 'javascript', because I thought learners would prefer a short version. Feel free to change the order if you think otherwise.

Reply:
Could this potentially lead to confusion about the exact command they need to enter?

I think it would be clearer to keep the original commend but then just add a comment that you can use `js` instead of javascript as a short form version.

Reply:
I see your point. 

How about removing 'javascript' add that 'js' is short for javascript in the above paragraph? I'm just seeing 'js' being more useful than a verbose 'javascript'.

Reply:
Yes, that would work too. Should I make another commit to this PR with your suggestion?

Sorry for the multiple replies. I thought I was deleting another review I accidentally created but turns out it was my initial reply to your comment.

Reply:
Yes just update your branch and then push it up.